src/components/Element/Note.jsx

```diff
@@ -47,10 +69,14 @@ export function Note() {
             return <Button key={index}>{item}</Button>;
           })}
         </ul>
-        <Button>Add to Notes</Button>
+        <Button onClick={addItem}>Add to Notes</Button>
       </nav>
 
-      <NoteList />
+      <NoteList>
```

Comment:
Sorry I meant the NoteList component is it really needed? it is a ul tag. May be it would be better for it to it to do iteration while remaining stateless.